build.gradle

```diff
@@ -179,8 +174,8 @@ dependencies {
     compile group: 'org.mozilla', name: 'rhino', version:'1.7.10'
 
     compile group: 'org.xmlunit', name: 'xmlunit-core', version:'2.6.2'
-    implementation group: 'net.sourceforge.pcgen', name: 'PCGen-base', version:'1.0.170'
-    implementation group: 'net.sourceforge.pcgen', name: 'PCGen-Formula', version:'1.0.195'
+    compile group: 'net.sourceforge.pcgen', name: 'PCGen-base', version:'1.0.170'
+    compile group: 'net.sourceforge.pcgen', name: 'PCGen-Formula', version:'1.0.195'
```

Comment:
Was this change intentional?

Reply:
Yes.  The issue is that with them being marked implementation, they do not get copied over for inclusion into the package.  So ClassNotFound exceptions when running the app. (incidentally, with the resources not being in src/main/resource, they do not get copied over either, and I had to add steps to do so).

I'm not a gradle expert, and their documentation leaves a lot to be desired. The documentation talks about exposing the dependencies to consumer classpaths; I've been wringing Java for a couple of decades, and have no idea what they mean by that.  With maven, things can be marked for inclusion to the product without being added to the classpath, but, that doesn't seem to be what this means.

Anyway, another option would be going with a fat jar file, so that everything ends up in one jar; maybe then the api & implementation vs. compile would make more sense?

Reply:
yes please, though note #5072. I've been (slowly) working on improving our build. It is also a moving target as gradle makes changes.

Reply:
actually, I need to understand the implication a little better: the documentation for these two is primarily around android. In the meantime leave them alone as `compile.`